src/esp/gfx/Drawable.h

```diff
@@ -24,6 +26,24 @@ class DrawableGroup;
  */
 class Drawable : public Magnum::SceneGraph::Drawable3D {
  public:
+  /** @brief Flag
+   * It will not be used directly in the base class "Drawable" but
+   * it will be used in a couple of sub-classes
+   */
+  enum class Flag {
+    /**
+     * indicates the mesh data has tangent attribute
+     */
+    HasTangent = 1 << 0,
```

Comment:
There are two conventions in the code base for enum. One is "all upper cases" style (e.g., sensor), and the other is "camel case" style (e.g., in DepthUnprojection, RenderCamera, Magnum).
I believe camel case is a bit better in terms of readability. So decided to use this style from now on.

Reply:
This decision is compatible with google c++ style guide on [enum naming](https://google.github.io/styleguide/cppguide.html#Enumerator_Names).  Do we want to use the style guide constant convention (i.e. prefix with "k") ?

Reply:
Definitely no Hungarian Notation or `k` prefixes, **please** :scream: 

We use C++11 scoped enums extensively (Google didn't use them much last time I checked), so I don't think prefixing the values with `k` makes any sense -- you already see it's an enum in `SomeEnum::SomeValue`, no need to add noise to it by doing `SomeEnum::kSomeValue`.

About casing I can't say either way -- who's responsible for coding style here? :) (Actually, sorry about `DepthUnprojection`, there I forgot to use the `UPPER_CASE`.) In Magnum I'm using `CamelCase` for everything except macros, but in Magnum's Python bindings I'm using `UPPER_CASE` again to be consistent with Python itself.

Reply:
No specific reasons. The legacy code can be refactored to use the `PascalCase` (almost the same as `camelCase`) as well. 
Note: Facebook has its own C++ coding conventions, style guidelines internally. We are following it. About `enum`, we will use `PascalCase` from now on as the guidance suggests.

The guidance also links to this page http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Renum-caps. 
It says do not use ALL_CAPS for enumerators to avoid clashes with macros.